test_lib/scylla_bench_tools.py

```diff
@@ -34,7 +32,5 @@ def create_scylla_bench_table_query(compaction_strategy=None, seed: int = None):
                     AND min_index_interval = 128
                     AND read_repair_chance = 0.0
                     AND speculative_retry = 'NONE'
-                    {compaction_strategy_option};
-                    """
-    LOGGER.debug("Generated a create-table query with a seed of [%s]: %s", seed, scylla_bench_table_query)
+                    """ + compaction_strategy_option
```

Comment:
maybe we should put it back in as f-string ?

Reply:
It is revert "as is".